apps/api/src/bootstrap.ts

```diff
@@ -121,7 +122,8 @@ export async function bootstrap(expressApp?): Promise<INestApplication> {
   await setupSwagger(app);
 
   Logger.log('BOOTSTRAPPED SUCCESSFULLY');
-
+  app.useGlobalFilters(new AllExceptionsFilter(app.get(PinoLogger)));
+  Logger.log('Filter Applied');
   if (expressApp) {
```

Comment:
The successful bootstrap log should be the last one we see to indicate the app has fully started up - if filter application fails, we could be mislead that the application started when it failed right at the end.

chore: Move these 2 newly added lines above the bootstrap message